### PR TITLE
Linter

### PR DESCRIPTION
**Pull request by yndx-birman**

### CODE REVIEW

**Comment by yndx-birman** [comment]
Need to merge first https://github.com/yandex-cloud/yfm-transform/pull/84

File: src/index.ts [hunk 1]
```diff
@@ -143,6 +149,7 @@ async function main() {
         const outputBundlePath: string = join(outputFolderPath, BUNDLE_FOLDER);
         const pathToConfig = _yargs.argv.config || join(_yargs.argv.input, '.yfm');
         const pathToRedirects = join(_yargs.argv.input, REDIRECTS_FILENAME);
+        const pathToLintConfig = join(_yargs.argv.input, LINT_CONFIG_FILENAME);
```
Comment (vovcyan) [inline src/index.ts:152]: Do we really need separate file for lint config? Can we combine it with `.yfm` ?

File: src/constants.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ export const TMP_OUTPUT_FOLDER = '.tmp_output';
 export const MAIN_TIMER_ID = 'Build time';
 export const SINGLE_PAGE_FOLDER = '_single_page';
 export const REDIRECTS_FILENAME = 'redirects.yaml';
+export const LINT_CONFIG_FILENAME = 'yfmlint.yaml';
```
Comment (vovcyan) [inline src/constants.ts:29]: .yfm-lint ?

File: src/index.ts [hunk 3]
```diff
@@ -143,6 +149,7 @@ async function main() {
         const outputBundlePath: string = join(outputFolderPath, BUNDLE_FOLDER);
         const pathToConfig = _yargs.argv.config || join(_yargs.argv.input, '.yfm');
```
Comment (vovcyan) [inline src/index.ts:150]: Let's move `.yfm` name into constants too

File: src/constants.ts (hunk 2 above)
Comment (yndx-birman) [reply src/constants.ts:29]: Why don't we want to use the yaml extension if `.yfm` includes yaml syntax?  `.yfm` includes yaml syntax and it's not highlighted in IDEs as yaml

Comment (vovcyan) [reply src/constants.ts:30]: That's true, but this approach separates hidden service files started with dot from documentation related files with `.yaml` extension.

**Pull request merged**